lib/settings/settings.dart

```diff
@@ -54,6 +55,13 @@ class Settings {
   set confirmMoves(bool val) =>
       prefs.setBool('$_behaviourKeyPrefix.confirm_moves', val);
 
+  String get confirmMovesBoardSize =>
```

Comment:
Let's better store the board size as an `int`, to avoid the parsing, and since the `NxN` format is just a presentation detail (e.g. we might want to change it later depending on localization, etc.).